server/plugin.go

```diff
@@ -3,11 +3,28 @@ package main
 import (
 	"fmt"
 	"net/http"
+	"strings"
 	"sync"
 
+	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mattermost-server/v5/plugin"
 )
 
+var generatedKeyValues = []string{
+	"token_secret",
+	"rsa_key",
+}
```

Comment:
These two keys in the jira plugin's kvstore do not marshal well to JSON. `token_secret` is an array of random bytes. `rsa_key` is an object that contains very large ints that get changed to scientific notation in JSON. I've made it so these two keys are converted to/from base64 to preserve their exact bytes.

There should be a more compatible solution, but this is a WIP.